Uno.Gallery/Uno.Gallery.Shared/Views/NestedPages/MaterialTopBarSampleNestedPage.xaml

```diff
@@ -42,32 +42,32 @@
 			<Grid x:Name="Home" Visibility="Collapsed">
 				<Border Background="LightGray">
 					<StackPanel HorizontalAlignment="Center" VerticalAlignment="Center">
-						<FontIcon Glyph="{StaticResource HomeIcon}" />
-						<TextBlock Text="Home" TextAlignment="Center" />
+						<FontIcon  Foreground="{ThemeResource OnSurfaceBrush}"  Glyph="{StaticResource HomeIcon}" />
+						<TextBlock Foreground="{ThemeResource OnSurfaceBrush}" Text="Home" TextAlignment="Center" />
 					</StackPanel>
 				</Border>
 			</Grid>
 			<Grid x:Name="Search" Visibility="Collapsed">
 				<Border Background="LightGray">
 					<StackPanel HorizontalAlignment="Center" VerticalAlignment="Center">
-						<FontIcon Glyph="{StaticResource SearchIcon}" />
-						<TextBlock Text="Search" TextAlignment="Center" />
+						<FontIcon  Foreground="{ThemeResource OnSurfaceBrush}" Glyph="{StaticResource SearchIcon}" />
+						<TextBlock Foreground="{ThemeResource OnSurfaceBrush}" Text="Search" TextAlignment="Center" />
 					</StackPanel>
 				</Border>
 			</Grid>
 			<Grid x:Name="Support" Visibility="Collapsed">
 				<Border Background="LightGray">
```

Comment:
This Background needs to be set to `{ThemeResource SurfaceBrush}` instead of `LightGray`